### DISCUSSION TITLE
Discussion: Why do we have same size for Header and Description of TextBox controls

### DISCUSSION THREAD

**Discussion by ssaran046** [microsoft/microsoft-ui-xaml#7452] [General]
# Discussion: Can we use a different size for the Header and Description?

Having the same size for the Header and Description seems odd. So we can use a different size for description as like the below image,

![image](https://user-images.githubusercontent.com/24865096/130024494-3ea21e7e-68dd-4cbc-98b9-fa2c03029cf6.png)

NumberBox UI: It has the same size for header and description

![image](https://user-images.githubusercontent.com/24865096/130025455-68efd747-7f37-49d6-b287-e60f7bdb12a4.png)

## Related Links

https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield

**Comment by ssaran046** [comment]
@Poopooracoocoo, It is a discussion. I will update the content.

**Comment by StephenLPeters** [comment]
I agree, I think the solution is to expose the description font properties as resources here: https://github.com/microsoft/microsoft-ui-xaml/blob/eaae2ce70bcbe4b00a9ed2724c7cdbf3f09d8765/dev/NumberBox/NumberBox.xaml#L279.

@ssaran046 this issue is only about numberbox right?

**Comment by StephenLPeters** [comment]
@teaP FYI

**Comment by ssaran046** [comment]
@StephenLPeters this issue exists on all the controls which have support for description.
